pages/faq.rst

```diff
@@ -5,61 +5,125 @@ Frequently asked questions
+We believe software should be open and accessible to all.  You should not have to pay to analyze your own data, no matter how much you have.
 
-Give it a try, you might be surprised!
+Graylog is licensed under the `GNU General Public License <http://www.gnu.org/licenses/gpl-3.0.en.html>`_.  We do not require license fees for production or non-production use.
 
-I already tried to use Elasticsearch for log management and it did not work well
---------------------------------------------------------------------------------
+How long do you support older versions of the Graylog product?
+--------------------------------------------------------------
+
+For our commercial support customers, we support older versions of Graylog up to 12 months after the next major release is available. So if you’re using 1.X, you will continue to receive 1.X support up to a full year after 2.0 has been released.
+
+Architecture
+============
+
+Isn’t Java slow? Does it need a lot of memory?
+-----------------------------------------------
 
-Sorry to hear that. The good news: Graylog is working around a lot of the
-log management specific shortcomings of Elasticsearch. Don't get us wrong:
-Elasticsearch is a great product! It just needs some special handling by our
-`graylog-server` process that you do not get if you are directly writing your logs
-and reading information from Elasticsearch. Especially the journalling of Graylog
-shields Elasticsearch against overloading and failing in weird ways.
+This is a concern that we hear from time to time. We understand Java has a bad reputation from slow and laggy desktop/GUI applications that eat a lot of memory. However, we are usually able to prove this assumption wrong. Well written Java code for server systems is very efficient and does not need a lot of memory resources.
+
+Give it a try, you might be surprised!
 
 What is MongoDB used for?
 -------------------------
 
-The MongoDB dependency of Graylog is there to store metadata that is not log data.
-None of your messages is ever stored in Graylog but for example user information
-or stream rules are. This is why you should not expect much load on MongoDB and
-thus don't have to worry too much about scaling it. It will just run aside your
-`graylog-server` processes and take almost no resources in our recommended setup
-architectures.
+Graylog uses MongoDB to store your configuration data, not your log data. Only metadata is stored, such as user information or stream configurations. None of your log messages are ever stored in MongoDB. This is why MongoDB does not have a big system impact, and you won’t have to worry too much about scaling it. With our recommended setup architecture, MongoDB will simply run alongside your graylog-server processes and use almost no resources.
+
+We have plans to introduce a database abstraction layer in the future. This will give you the flexibility to run MongoDB, MySQL or any other database to store metadata.
```

Comment:
We should mention it because it is a concern of many users and we can show that we are aware of this being not optimal. It is good to have it in there as long as we don't mention *when*.